jupyter-docker/user-scripts/enable_download_as_pdf.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Oh there it is. :) Is this used?

Reply:
Used in the test?
I uploaded the script to a bucket and used it to test pdf download manually, it worked fine.